considerations.mkd

```diff
@@ -4,13 +4,17 @@ GeoJSON shares security issues common to all JSON content types.  See
 [RFC7159] Section 12 for additional information. GeoJSON does not
 provide executable content.
 
+GeoJSON does not provide privacy or integrity services.  If sensitive
+data requires privacy or integrity protection, those must be provided by
+the transport — for example TLS or HTTPS. There will be cases in which
+stored data need protection, which is out of scope for this document.
+
 As with other geographic data formats, e.g., [KMLv2.2], providing
 details about the locations of sensitive persons, animals, habitats, and
-facilities can expose them to unauthorized tracking or injury.  GeoJSON
-does not provide privacy or integrity services.  If sensitive data requires
-privacy or integrity protection, those must be provided by the transport — for
-example TLS or HTTPS. There will be cases in which stored data need protection,
-which is out of scope for this document.
+facilities can expose them to unauthorized tracking or injury. Data
```

Comment:
I like the sentiment, but feel that the wording is a bit confusing. How about:

"Data providers should recognize the risk of inadvertantly identifying individuals if anonymized datasets are not adequately skewed or locations are not sufficiently fuzzed [Sweeney]."